app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -2201,44 +2190,44 @@ class ChatActivity :
             ConversationUtils.isLobbyViewApplicable(currentConversation!!, spreedCapabilities)
         ) {
             if (shouldShowLobby()) {
-                binding.lobby.lobbyView.visibility = View.VISIBLE
-                binding.messagesListView.visibility = View.GONE
-                binding.fragmentContainerActivityChat.visibility = View.GONE
-                binding.progressBar.visibility = View.GONE
-
-                val sb = StringBuilder()
-                sb.append(resources!!.getText(R.string.nc_lobby_waiting))
-                    .append("\n\n")
-
-                if (currentConversation?.lobbyTimer != null &&
-                    currentConversation?.lobbyTimer !=
-                    0L
-                ) {
-                    val timestampMS = (currentConversation?.lobbyTimer ?: 0) * DateConstants.SECOND_DIVIDER
-                    val stringWithStartDate = String.format(
-                        resources!!.getString(R.string.nc_lobby_start_date),
-                        dateUtils.getLocalDateTimeStringFromTimestamp(timestampMS)
-                    )
-                    val relativeTime = dateUtils.relativeStartTimeForLobby(timestampMS, resources!!)
-
-                    sb.append("$stringWithStartDate - $relativeTime")
-                        .append("\n\n")
-                }
-
-                sb.append(currentConversation!!.description)
-                binding.lobby.lobbyTextView.text = sb.toString()
+                showLobbyView()
             } else {
                 binding.lobby.lobbyView.visibility = View.GONE
                 binding.messagesListView.visibility = View.VISIBLE
-                checkShowMessageInputView()
+                binding.fragmentContainerActivityChat.visibility = View.VISIBLE
             }
-        } else {
```

Comment:
As this else branch was removed.. did you make sure this won't introduce any bug?